modules/users/server/tests/user-auth.controller.unit.js

```diff
@@ -82,14 +83,14 @@ describe('UserAuthController', () => {
     to: 'test@example.com',
     from: 'don\'t care',
     subject: 'Verification Email',
-    text: 'Verify your email by going here: http://' + mockConfig.app.host + ':' + mockConfig.app.port_http + '/verifyEmail;token=' + token
+    text: 'Verify your email by going here: http://' + mockConfig.app.host + '/verifyEmail;token=' + token
```

Comment:
Good todo item...  Pull this from a template instead so it's not a straight one-line text email.